data/hai/culture hai.txt

```diff
@@ -0,0 +1,235 @@
+		distance 1 3
+		government "Hai"
+	to offer
+		random < 1
+	on offer
+		conversation
+			`As you are walking through the spaceport, you come across the uncommon pairing of a Hai and a human, their conversation standing out by virtue of speaking the common tongue of humanity.`
+			`	"Oh, oh, Gemanari, this looks interesting!" The human woman looks to be in her late sixties and, when you turn to look, seems to be clucking over a holographic advertisement for a theater show. "This looks adorable, doesn't it? Oh, and it says they're using ripple ray refractive holographic flight effects!"`
+			`	The advertisement does indeed look very impressive and you pause to observe for a moment.`
+			`	The Hai, presumably Gemanari, also appears to be a more mature lady who squints at it and wrinkles her nose. "Ugh, that's the Germontian interpretation though."`
+			`	This receives a puzzled look to which Gemanari thinks for a second before clarifying, "You would say the Social Darwinist interpretation."`
+			`	The lady grimaces and makes like she's vomiting, "I can't believe you still have those schools of thought in a society this old."`
+			`	"Oh we don't really, at least not in scientific circles, but a compelling story will outlive such things. I can't stand it personally but it is a classic." When the lady looks unimpressed Gemanari titters and takes her gently by the arm to steer her away again. "Don't worry Alinta, I think the Misheep Wah'uden company is running my preferred interpretation on <destination> at the moment. We can see if they have any cheap tickets. Of course my aunt would insist that the hundred and eighty-third version is the only real one but she always..."`
+			`	You lose the conversation as they trail away from you, leaving you to wonder just what was so special about the hundred and eighty-third version of a play about a prehistoric flying Hai found frozen in the ice.`
+				decline
+
+mission "ZZCU Musical Entertainment Expo"
+	minor
+	source
+		government "Hai"
+		not attributes "station"
+	to offer
+		random < 1
+	on offer
+		conversation
+			`While stopped in the spaceport waiting for refueling you are surprised by how hard it is to find somewhere to sit and have a meal & a drink. It seems Hai have crammed themselves into every bar or similar venue with a public display screen, arrayed in sashes and mantles and all manner of other paraphernalia which, near as you can tell, looks like... team sport merchandise?`
+			`	You're not really an expert on exactly what such things look like in Hai society and so when you finally find somewhere to sit, at a bar for lack of any other options, you ask the tired looking bartender what exactly is going on.`
+			`	"Oh this? They're all here for the Musical Entertainment Expo streams. Unfortunately, none of them start drinking until at least one of their teams has announced something of worth," he says with an exasperated eye-roll.`
+			choice
+				`	"Sure, can I just get some food here?"`
```

Comment:
The `Sure` feels a little out of place to me because the First Last isn't really affirming anything.
```suggestion
				`	"Okay. Can I just get some food here?"`
```
There's a lot of other alternatives you could do too.